app/src/main/java/com/rockthevote/grommet/ui/registration/InvalidRegistrationException.kt

```diff
@@ -0,0 +1,12 @@
+package com.rockthevote.grommet.ui.registration
+
+import androidx.annotation.StringRes
+
+/**
+ * Provided registration data is invalid.
+ */
+class InvalidRegistrationException(
```

Comment:
there's some non-standard server response code logic i'll have to dig up and share when we get to that part.